test/blackbox-tests/test-cases/mdx-stanza/paths.t

```diff
@@ -13,12 +13,14 @@ Absolute paths cause an error.
   > ```
   > EOF
 
-  $ dune runtest 2>&1 | awk '/Internal error/,/Raised at/'
-  Internal error, please report upstream including the contents of _build/log.
-  Description:
-    ("Local.relative: received absolute path",
-    { t = "."; path = "/etc/passwd" })
-  Raised at Stdune__Code_error.raise in file "otherlibs/stdune/code_error.ml",
+  $ dune runtest
+  File "dune", line 1, characters 0-5:
+  1 | (mdx)
+      ^^^^^
+  Error: Paths referenced in mdx files must be relative. This stanza refers to
```

Comment:
Isn't it at least possible to tell the user which mdx file is at fault?

Reply:
Unfortunately, no. The stanza calls `ocaml-mdx deps <files in stanza>` so it's a file list -> file list interface with no precise mapping. We plan to extend the output format to add locs to these strings to get a better error message.

Reply:
done. let me know what you think